src/GitUtils.ps1

```diff
@@ -557,6 +610,19 @@ function Add-SshKey([switch]$Quiet) {
         }
 
         if ($args.Count -eq 0) {
+            # Win10 ssh agent will prompt for key password even if the key has already been added
```

Comment:
How do we get here for Win10 ssh agent?  Won't the `return` on line 507 result in this never being executed?

Reply:
This is in `Add-SshKey`.

Reply:
D'oh!  Yeah, that was kind of obscured by GH's helpful collapsing of unchanged lines.  :-)